### DISCUSSION TITLE
请问v5版本可以使用在生产环境吗？

### DISCUSSION THREAD

**Discussion by zdawei** [ant-design/ant-design-mobile#3894] [General]
想问下v5可以使用在生产环境吗？

**Comment by xiaohuoni** [comment]
还没开发完呢，不急。稳住发育，兄弟

**Comment by xiaohuoni** [comment]
还没开发完呢，不急。稳住发育，兄弟

**Comment by awmleer** [comment]
可以参考下[文档](https://next.mobile.ant.design/guide/faq#%E6%88%91%E5%BA%94%E8%AF%A5%E7%8E%B0%E5%9C%A8%E5%BC%80%E5%A7%8B%E4%BD%BF%E7%94%A8-v5-%E7%89%88%E6%9C%AC%E5%90%97%EF%BC%9F)上的说明～

![image](https://user-images.githubusercontent.com/16526078/129130643-c5260914-0bb4-4e6a-aa75-c42180312014.png)

在蚂蚁内部，我们是已经在生产环境开始使用 v5 版本了，但是这是基于我们会确保及时跟进版本升级的前提，所以还是要根据具体情况选择要不要现在开始使用哈

**Comment by jkiss** [comment]
测试过了，就能用 :)

**Comment by zdawei** [comment]
感谢大佬~
已经在使用中~

**Comment by Me-UserName** [comment]
没有日历组件？

**Reply by awmleer** [reply]
日历组件后面会做，目前可以先用 DatePicker 作为替代

**Comment by lxtlxy** [comment]
请问支持主题定制吗？

**Reply by awmleer** [reply]
#3904
